### PR TITLE
fixed pagination for better customisability.

### PR DESCRIPTION
**Pull request by iham**

### CODE REVIEW

File: plone/app/layout/nextprevious/nextprevious.pt [hunk 1]
```diff
@@ -18,10 +18,10 @@
 
         <li class="previous" tal:condition="previous">
           <a title="Go to previous item"
-             i18n:attributes="title title_previous_item"
-             tal:attributes="href previous/url">
-            «
-            <span i18n:translate="label_previous_item" tal:omit-tag="">
+              i18n:attributes="title title_previous_item"
+              tal:attributes="href previous/url">
+            <span class="arrow"></span>
```
Comment (ale-rt) [inline plone/app/layout/nextprevious/nextprevious.pt]: Can we leave the « (or &laquo;) inside the span?

Comment (iham) [reply plone/app/layout/nextprevious/nextprevious.pt]: hi @ale-rt,

i did some refactoring of that template to achieve better customisability using css.
for example:
on a mobile device like a smart phone the label-text gets pretty long, or better to say: uses a lot of space.

as i remember the arrows were realised using css :before and :after, but when hiding the labels you also loose the arrows.

you still can put the < or « into a custom template and remove the css-part. but i would not recommend it as a default.

this is part of this fix:
https://github.com/plone/plonetheme.barceloneta/pull/151

any other/better ideas?

Comment (djowett) [reply plone/app/layout/nextprevious/nextprevious.pt]: Hmm, I did something similar overriding a plone.batching template last week. That has &laquo; and &raquo; in the template - which is clearer than the empty 'arrows' span. But then &laquo; might look a bit 'old' beside more modern icons. However, @iham will this only work for barceloneta?

Comment (agitator) [reply plone/app/layout/nextprevious/nextprevious.pt]: you can always add those arrows via css content: "»";
idea is no need to customize the template

Comment (iham) [reply plone/app/layout/nextprevious/nextprevious.pt]: sadly, its quiet granular ...
the css part is in the barceloneta theme
the html is in plone.app.layout AND in plone.batching

Comment (agitator) [reply plone/app/layout/nextprevious/nextprevious.pt]: @ale-rt ping?

**Comment by ale-rt** [comment]
Test failure seems unrelated, @jensens @agitator is it ok to merge this anyway?

**Pull request merged**